### PR TITLE
cleanup edge aggregation where input

### PR DESCRIPTION
**Pull request by Andy2003**
# Description

name *EdgeAggregationWhereInput by its type

## Complexity

Complexity: Low

# Checklist

The following requirements should have been met (depending on the changes in the branch):

- [ ] Documentation has been updated
- [x] TCK tests have been updated
- [ ] Integration tests have been updated
- [ ] Example applications have been updated
- [ ] New files have copyright header
- [x] CLA (https://neo4j.com/developer/cla/) has been signed

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
Thanks for the Neo4j GraphQL Toolbox updates.

The Neo4j GraphQL Toolbox has now been torn down - reopening this PR will republish it.

File: packages/graphql/src/schema-model/relationship/model-adapters/RelationshipDeclarationOperations.ts [hunk 1]
```diff
@@ -275,6 +275,12 @@ export class RelationshipDeclarationOperations {
         )}EdgeSort`;
     }
 
+    public get edgeAggregationWhereInputTypeName(): string {
+        return `${upperFirst(this.relationshipDeclaration.source.name)}${upperFirst(
```
Comment (a-alle) [inline packages/graphql/src/schema-model/relationship/model-adapters/RelationshipDeclarationOperations.ts:269]: The `upperFirst` here is not consistent with the rest of the type names

**Review by a-alle — CHANGES_REQUESTED** [review]
I see some more schema tests need to be updated as well

File: packages/graphql/src/schema-model/relationship/model-adapters/RelationshipDeclarationOperations.ts (hunk 1 above)
Comment (Andy2003) [reply packages/graphql/src/schema-model/relationship/model-adapters/RelationshipDeclarationOperations.ts:269]: I know but they are inconstistent for all the edge types:

https://github.com/neo4j/graphql/blob/931782b4aeb943cf5951cbd692822f03b72a9ae7/packages/graphql/src/schema-model/relationship/model-adapters/RelationshipDeclarationOperations.ts#L247-L276

I think they all need to be  without the 1st  `upperFirst`,  what do you think @a-alle ?

Comment (Andy2003) [reply packages/graphql/src/schema-model/relationship/model-adapters/RelationshipDeclarationOperations.ts:269]: I create a separate PR for this

**Review by a-alle — APPROVED** [review]

**Pull request merged**